holo-bgp/tests/packet/ip.rs

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) The Holo Core Contributors
+//
+// SPDX-License-Identifier: MIT
+
+// Meant for the ipv4/6 encoders and decoders inside the
+// holo_bgp::packet::message file.
+
+use bytes::Bytes;
+use holo_bgp::packet::error::{DecodeError, UpdateMessageError};
+use holo_bgp::packet::message::{decode_ipv4_prefix, decode_ipv6_prefix};
+
+// Try decode_ipv6_prefix with empty bytes, which should not be decodable.
+#[test]
+fn test_decode_ipv6_prefix1() {
```

Comment:
I don't think this new file is necessary. Since IP prefixes are only present in Update messages, these two tests could probably be moved to the *update.rs* file.

Reply:
Done.